File: baekjoon/bronze/level1/test_1546/Main.java
package bronze.level1.test_1546;

import java.io.*;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int n = Integer.parseInt(br.readLine());
        int[] input = Arrays.stream(br.readLine().split(" ")).mapToInt(Integer::parseInt).toArray();
        double sum = 0;

        Arrays.sort(input);

        for(double i : input) {
            sum += i / input[n - 1] * 100;
        }

        bw.write(String.valueOf(sum / n));
        bw.close();
    }
}

File: baekjoon/bronze/level1/test_1546/kotlin/Main.kt
package baekjoon.bronze.level1.test_1546.kotlin

fun main() {
    val n = readln().toInt()
    var scores = readln().split(" ").map { it.toDouble() }

    scores = scores.sorted()
    var sum = 0.0
    for (i in 0..<n) {
        sum += scores[i] / scores[n - 1] * 100
    }

    print(sum / scores.size)
}
